recipe/build_base.sh

```diff
@@ -508,7 +508,8 @@ fi
 rm -rf ${PREFIX}/lib/python${VER}/distutils/command/*.exe
 
 python -c "import compileall,os;compileall.compile_dir(os.environ['PREFIX'])"
-rm ${PREFIX}/lib/libpython${VER}.a
+rm ${PREFIX}/lib/libpython${VERABI}.a
+
```

Comment:
I wonder if it also makes sense to
```
ln -s ${PREFIX}/lib/libpython${VER}d.so ${PREFIX}/lib/libpython${VER}.so
```

I suggest this because gdb complaints about the following:
```
gdb: error while loading shared libraries: libpython3.10.so.1.0: cannot open shared object file: No such file or directory
```